src/main/java/use_case/Reschedule.java

```diff
@@ -0,0 +1,26 @@
+package main.java.use_case;
+
+import main.java.entity_gateway.CalendarManager;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class Reschedule implements RescheduleAuto{
```

Comment:
Having a noun as the class name would be better. Maybe something like `Rescheduler` and `AutoRescheduler`